conda_devenv/devenv.py

```diff
@@ -142,7 +142,7 @@ def render_activate_script(environment, shell):
                 value = pathsep.join(value) + pathsep + "%{variable}%".format(variable=variable)
 
             script.append("set CONDA_DEVENV_BKP_{variable}=%{variable}%".format(variable=variable))
-            script.append("set \"{variable}={value}\"".format(variable=variable, value=value))
+            script.append("set {variable}={value}".format(variable=variable, value=value))
```

Comment:
We should not remove the quotes. And actually we need to add quotes for all variables in all shells.
Maybe we create an issue to fix that?